ale/drivers/base.py

```diff
@@ -339,50 +420,39 @@ def center_ephemeris_time(self):
         """
         if not hasattr(self, '_center_ephemeris_time'):
             halflines = self.image_lines / 2
-            center_sclock = self.starting_ephemeris_time + halflines * self._exposure_duration
+            center_sclock = self.starting_ephemeris_time + halflines * self.line_exposure_duration
             self._center_ephemeris_time = center_sclock
         return self._center_ephemeris_time
 
-    @property
-    def t0_ephemeris(self):
-        return self.starting_ephemeris_time - self.center_ephemeris_time
-
-    @property
-    def t0_quaternion(self):
-        return self.starting_ephemeris_time - self.center_ephemeris_time
 
+class Isis3():
     @property
-    def dt_ephemeris(self):
-        return (self.ending_ephemeris_time - self.starting_ephemeris_time) / self.number_of_ephemerides
+    def start_time(self):
+        return self._label['IsisCube']['Instrument']['StartTime']
```

Comment:
Does it make sense to do this or to recursively just search for 'StartTime' in the whole label? I think `plio` has the `find_in_dict` func.

Reply:
This is one of the keys that is explicitly required by ISIS to be at this location in the label. I think this is fine.

Reply:
Do we know if this is the case for all of the ISIS cube specific labels?